puppeteer-tests/src/utils.ts

```diff
@@ -18,8 +18,8 @@ export const setupBrowser = async (
     executablePath: process.env.BROWSER,
   })
   const page = await browser.newPage()
-  await page.setDefaultNavigationTimeout(120000)
-  await page.setDefaultTimeout(defaultTimeout)
+  page.setDefaultNavigationTimeout(120000)
+  page.setDefaultTimeout(defaultTimeout)
```

Comment:
Why are we no longer `await`ing these?

Reply:
They don't return a promise, I had a warning saying it wasn't doing anything.